libvcs4j-tools/src/main/java/de/unibremen/informatik/st/libvcs4j/spoon/SpoonModelBuilder.java

```diff
@@ -0,0 +1,459 @@
+package de.unibremen.informatik.st.libvcs4j.spoon;
```

Comment:
This class was renamed to [EnvironmentBuilder](https://github.com/uni-bremen-agst/libvcs4j/blob/master/libvcs4j-tools/src/main/java/de/unibremen/informatik/st/libvcs4j/spoon/EnvironmentBuilder.java). Please remove it.